### PR TITLE
Adding functionality for Amazon Linux

### PR DESCRIPTION
**Pull request by sladd0175**
The current version does not support Amazon Linux.  I have added the necessary pieces

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-java",
-  "version": "1.4.3",
+  "version": "1.4.4",
```
Comment (hunner) [inline metadata.json]: Please don't change the version as part of a non-release PR; thanks.

**Comment by bmjen** [comment]
Thanks for the contribution @sladd0175. Can you fix the unit test failures as well as address the updates in metadata.json @hunner mentioned?

**Comment by tphoney** [comment]
@sladd0175 do you need any help with this ?

**Comment by HelenCampbell** [comment]
Hey @sladd0175 , thank you for your contribution, however due to the amount of work needed done and the lack of activity I am going to close this PR. Feel free to resubmit, we will happily review another PR :) Thanks again!

**Pull request closed without merging**